docs/webhooks/create-incoming-webhook.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Create an Incoming Webhook
+intro: Incoming Webhooks are a simple way to integrate external services with your Devopness environments, using a unique URL through which a JSON payload can be sent to run a pipeline with custom input data. Some use cases of external services events include "deploy applications automatically based on GitHub/GitLab/Bitbucket repository updates", "provision and scale infrastructure resources when you payment gateway confirms that a custom subscription has been processed", and many more use cases that can benefit from webhooks integration. Learn how to create an incoming webhook to run pipelines programmatically when events happen on external services.
+links:
+    overview:
+    quickstart:
+    previous:
+    next:
+    guides:
+    related:
+    featured:
+---
+
+Devopness is an API-First product, meaning that every feature in our product is first designed with the API usage in mind. That development approach implies that every action you make using our web app can also be directly made using API calls.
+
+Webhooks, for now, are an API only feature; so this post will guide you through the usage of our API to help you create a webhook.
+
+To create our incoming webhook, we need the `Application ID` (`<application_id>`).
+
+<details open>
```

Comment:
Here we are repeating what is already documented in [docs/applications/deploy-application-using-incoming-hook.md](https://github.com/devopness/devopness/pull/674/files#diff-a0aede2bc4c3be7826b0f00dc555e8c59b9662ccabc4a9c327e3c62f0b3d73be).
Could we just add a link here, using the same pattern using on SSL/DNS to point the reader to the `Find server IP address`?

Altough that creates a cyclic dependency between both `deploy application` and `create webhook` docs, I can't see any problems on that approach as it will help us ensure our documentation is more concise and less redundant.

What do you think?

Reply:
I agree with the change